rfqmsg/messages.go

```diff
@@ -17,8 +21,16 @@ func (id ID) String() string {
 	return hex.EncodeToString(id[:])
 }
 
-// SerialisedScid is a serialised short channel id (SCID).
-type SerialisedScid uint64
+// Scid returns the short channel id (SCID) of the RFQ message.
+func (id ID) Scid() SerialisedScid {
+	// Given valid RFQ message id, we then define a RFQ short chain id
+	// (SCID) by taking the last 8 bytes of the RFQ message id and
+	// interpreting them as a 64-bit integer.
+	scidBytes := id[24:]
```

Comment:
nit
```suggestion
	scidBytes := id[len(id)-8]
```

Reply:
I don't think your suggesting works. We want to select the last 8 elements from the `id` array, not the single element at `len(id)-8`.